en/simulation/gazebo.md

```diff
@@ -236,90 +257,72 @@ This thin wrapper simply embeds app arguments like current working directories a
 To start Gazebo and PX4 separately:
 
 * Run gazebo (or any other sim) server and client viewers via the terminal:
-  ```
+  ```sh
   make px4_sitl gazebo_none_ide
   ```
 * In your IDE select `px4_<mode>` target you want to debug (e.g. `px4_iris`)
 * Start the debug session directly from IDE
 
-This approach significantly reduces the debug cycle time because simulator (e.g. gazebo) is always running in background and you only re-run the px4 process which is very light.
+This approach significantly reduces the debug cycle time because simulator (e.g. Gazebo) is always running in background and you only re-run the px4 process which is very light.
 
 
-## Video Streaming
+## Video Streaming {#video}
 
 PX4 SITL for Gazebo supports UDP video streaming from a Gazebo camera sensor attached to a vehicle model.
-You can connect to this stream from *QGroundControl* (on UDP port 5600) and view video of the Gazebo environment from the simulated vehicle - just as you would from a real camera.
-The video is streamed using a *gstreamer* pipeline.
+When streaming is enabled, you can connect to this stream from *QGroundControl* (on UDP port 5600) and view video of the Gazebo environment from the simulated vehicle - just as you would from a real camera.
+The video is streamed using a *gstreamer* pipeline and can be enabled/disabled using a button in the Gazebo UI.
+
+The Gazebo camera sensor is supported/enabled on the following frames:
+* [Typhoon H480](#typhoon_h480)
 
-> **Note** Video streaming from Gazebo and the Gazebo widget to turn streaming on/off are not enabled by default.
-This article explains how to enable them. In the near future we expect these features to be enabled by default.
 
 ### Prerequisites
 
-Ubuntu: Install *Gstreamer 1.0* and its dependencies:
-```
-sudo apt-get install $(apt-cache --names-only search ^gstreamer1.0-* | awk '{ print $1 }' | grep -v gstreamer1.0-hybris) -y
-```
+*Gstreamer 1.0* is required for video streaming.
+The required dependencies should already have been [installed when you set up Gazebo](#installation) (they are included in the standard PX4 installation scripts/instructions).
+
+For information only:
+- **Ubuntu:** Install *Gstreamer 1.0* and its dependencies:
+  ```
+  sudo apt-get install $(apt-cache --names-only search ^gstreamer1.0-* | awk '{ print $1 }' | grep -v gstreamer1.0-hybris) -y
```

Comment:
These dependencies are included in `ubuntu.sh`. Why not just link to that?

Reply:
@julianoes Essentially I do above all this by saying "this is all installed previously using standard installation - this is for info only". I figured it might be useful for people to see what it is that is needed. I could delete both this and the mac stuff if you think a good ideas?

Reply:
It's a very complicated command but yes :smile: it can be useful.

Reply:
OK, in that case I have done as you suggested and removed the command. However I've provided an FYI only list of the dependencies so they can check at least for those.